tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -996,7 +996,7 @@ Status IrEmitterUnnested::EmitRowReduction(
   //   for (shuffle_distance = 16; shuffle_distance > 0; shuffle_distance /= 2)
   //     partial_result = Reducer(
   //         partial_result,
-  //         __shfl_down(partial_result, shuffle_distance));
+  //         __shfl_down_sync(0xFFFFFFFF, partial_result, shuffle_distance));
```

Comment:
CUDA_WARP_ALL?

Reply:
Switched to CUDA_WARP_ALL. But if this is ever un-commented, a dependency on cuda_kernel_helper.h will need to be added.